### ISSUE TITLE
libfixmath_unittest test fails

### ISSUE THREAD

**Issue by jia200x** [RIOT-OS/RIOT#10208]
#### Description

The libfixmath unittests fail in `b-l072z-lrwan1` board. The test sends TIMEOUT.
I haven't tried with other boards yet. I will check.
#### Steps to reproduce the issue

`make flash test` on `b-l072z-lrwan1`.
#### Expected results

Test passes
#### Actual results

```
2018-10-19 16:54:14,624 - INFO # ----Testing fix16_to_str corner cases----
2018-10-19 16:54:14,626 - INFO # 1234.5678 = 1234.5678
2018-10-19 16:54:14,631 - INFO # OK: strcmp(buf, "1234.5678") == 0
2018-10-19 16:54:14,633 - INFO # -1234.5678 = -1234.5678
2018-10-19 16:54:14,637 - INFO # OK: strcmp(buf, "-1234.5678") == 0
2018-10-19 16:54:14,642 - INFO # OK: strcmp(buf, "0") == 0
2018-10-19 16:54:14,645 - INFO # OK: strcmp(buf, "1") == 0
2018-10-19 16:54:14,647 - INFO # (fix16_t)1 = 0.00002
2018-10-19 16:54:14,652 - INFO # OK: strcmp(buf, "0.00002") == 0
2018-10-19 16:54:14,654 - INFO # (fix16_t)-1 = -0.00002
2018-10-19 16:54:14,657 - INFO # OK: strcmp(buf, "-0.00002") == 0
2018-10-19 16:54:14,660 - INFO # (fix16_t)65535 = 0.99998
2018-10-19 16:54:14,664 - INFO # OK: strcmp(buf, "0.99998") == 0
2018-10-19 16:54:14,666 - INFO # (fix16_t)65535 = 1.0000
2018-10-19 16:54:14,670 - INFO # OK: strcmp(buf, "1.0000") == 0
2018-10-19 16:54:14,673 - INFO # fix16_maximum = 32767.99998
2018-10-19 16:54:14,677 - INFO # OK: strcmp(buf, "32767.99998") == 0
2018-10-19 16:54:14,680 - INFO # fix16_minimum = -32768.00000
2018-10-19 16:54:14,685 - INFO # OK: strcmp(buf, "-32768.00000") == 0
2018-10-19 16:54:14,686 - INFO # 
2018-10-19 16:54:14,689 - INFO # ----Testing fix16_from_str corner cases----
2018-10-19 16:54:14,696 - INFO # OK: fix16_from_str("1234.5678") == fix16_from_dbl(1234.5678)
2018-10-19 16:54:14,703 - INFO # OK: fix16_from_str("-1234.5678") == fix16_from_dbl(-1234.5678)
2018-10-19 16:54:14,710 - INFO # OK: fix16_from_str("   +1234,56780   ") == fix16_from_dbl(1234.5678)
2018-10-19 16:54:14,714 - INFO # OK: fix16_from_str("0") == 0
2018-10-19 16:54:14,719 - INFO # OK: fix16_from_str("1") == fix16_one
2018-10-19 16:54:14,724 - INFO # OK: fix16_from_str("1.0") == fix16_one
2018-10-19 16:54:14,729 - INFO # OK: fix16_from_str("1.0000000000") == fix16_one
2018-10-19 16:54:14,734 - INFO # OK: fix16_from_str("0.00002") == 1
2018-10-19 16:54:14,739 - INFO # OK: fix16_from_str("0.99998") == 65535
2018-10-19 16:54:14,745 - INFO # OK: fix16_from_str("32767.99998") == fix16_maximum
2018-10-19 16:54:14,752 - INFO # OK: fix16_from_str("-32768.00000") == fix16_minimum
2018-10-19 16:54:14,752 - INFO # 
2018-10-19 16:54:14,754 - INFO # ----Extended testing for whole range----
Timeout in expect script at "child.expect('SUCCESS', timeout=TIMEOUT)" (tests/libfixmath_unittests/tests/01-run.py:18)
```
#### Versions

```
Operating System Environment
-----------------------------
       Operating System: "Arch Linux" 
                 Kernel: Linux 4.18.5-arch1-1-ARCH x86_64 unknown

Installed compiler toolchains
-----------------------------
             native gcc: gcc (GCC) 7.2.1 20171128
      arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
                avr-gcc: missing
       mips-mti-elf-gcc: missing
             msp430-gcc: missing
   riscv-none-embed-gcc: missing
                  clang: clang version 7.0.0 (tags/RELEASE_700/final)

Installed compiler libs
-----------------------
   arm-none-eabi-newlib: "2.5.0"
    mips-mti-elf-newlib: missing
riscv-none-embed-newlib: missing
               avr-libc: missing (missing)

Installed development tools
---------------------------
                  cmake: cmake version 3.12.1
               cppcheck: Cppcheck 1.84
                doxygen: 1.8.14
                 flake8: missing
                    git: git version 2.18.0
                openocd: Open On-Chip Debugger 0.10.0+dev-00226-g1c2e3d41d (2018-01-02-18:28)
                 python: Python 3.7.0
                python2: Python 2.7.15
                python3: Python 3.7.0
             coccinelle: missing
```

**Comment by jia200x** [comment]
@jcarrano @aabadie could you check if this issue affects you as well?

**Comment by jia200x** [comment]
Solved with #10201 . Close issue.
